sdk/core/Azure.Core/src/Pipeline/ContentTypeUtilities.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+
+namespace Azure.Core.Pipeline
+{
+    internal class ContentTypeUtilities
+    {
+        public static bool IsText(string contentType, out Encoding encoding)
```

Comment:
Have you considered this as a method on the `Response` class directly?   It would seem to fit contextually and would avoid having a static utilities class.

Reply:
I did, but it's applied to request in this PR so I had to move it to a utility class.